src/main/java/com/redhat/jenkins/plugins/cachet/ResourceUpdater.java

```diff
@@ -87,6 +142,15 @@ public static void setResources() {
             } catch (IOException e) {
                 log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
                 rmap = null;
+            } catch (NoSuchAlgorithmException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
+            } catch (KeyStoreException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
+            } catch (KeyManagementException e) {
+                log.log(Level.SEVERE, "Unhandled exception retrieving Cachet component list.", e);
+                rmap = null;
```

Comment:
Why not just catch these all on the same line, like this?
    catch (IOException | NoSuchAlgorithmException | KeyStoreException | KeyManagementException e)
Then we can get rid of all the duplicated code....

Reply:
Yep. I will fix this mess.